packages/markdowndb/README.md

```diff
@@ -1,3 +1,109 @@
-# markdowndb
+# MarkdownDB [![](https://badgen.net/npm/v/@flowershow/markdowndb)](https://www.npmjs.com/package/@flowershow/markdowndb)
 
-Parse markdown files and store them in an SQL database.
+MarkdownDB is a ... that parses your markdown files and saves them to your local SQLite database, so that you can easily import them into your application.
+
+**🚧 MarkdownDB is in its early development stage**
+
+## How to use in Next.js app
```

Comment:
Good start. Personally i think i'd leave use in nextjs app second and have first a really simple intro like:

- create the index
- boot up sqlite and do some queries, find me all docs with these tags (i don't know if we have a demo db we can use)

Then tutorial 2 is: building a blog in nextjs (or even just: here's how you'd use it to get stuff in `getStaticProps`)

Reply:
NB: I see i already had an example like this in the body of the issue at #471